app/helpers/practices_helper.rb

```diff
@@ -3,6 +3,13 @@ def fetch_offices
     @office_data = JSON.parse(File.read("#{Rails.root}/lib/assets/practice_origin_lookup.json"))
     @office_data["departments"][0]["offices"].to_json
   end
+  def url_is_internal(s_url)
+    allowed_hosts = ['marketplace.va.gov', 'localhost:3200']
+    allowed_hosts.each do |ah|
+      return true if s_url.downcase().include?(ah)
+    end
+    return false
+  end
```

Comment:
This is really good Brad. Just a couple of things. Can we change the name to `is_url_internal?(s_url)`? This'll make it a little more clear that we're returning a boolean. Instead of looping through the array, we should just be able to call:
```
internal_urls.any? { |url| s_url.downcase.include?(url) }
```

so, the entire method might look something like:
```
def is_url_internal?(s_url)
   internal_urls = ['marketplace.va.gov', 'localhost:3200']
   internal_urls.any? { |url| s_url.downcase.include?(url) }
end
```

or, if you'd like to use a block:
```
def is_url_internal?(s_url)
   internal_urls = ['marketplace.va.gov', 'localhost:3200']
   internal_urls.any? do |url|
     s_url.downcase.include?(url)
   end
end
```

Here's a reference to the `.any?` method: https://apidock.com/ruby/Enumerable/any%3F

Reply:
Yeah.. I forgot about any?   I'm familiar with it.. First implementation ^^ I think is best.  Got that all fixed up and tested.